src/sentry/models/auditlogentry.py

```diff
@@ -13,6 +13,24 @@
 MAX_ACTOR_LABEL_LENGTH = 64
 
 
+def format_ondemand_budget(ondemand_data):
+    if type(ondemand_data) is dict:
+        budget_mode = ondemand_data.get("budgetMode", None)
+        if budget_mode == "shared":
+            shared_budget = ondemand_data.get("sharedMaxBudget", 0)
+            return f"shared on-demand of {format_ondemand_max_spend(shared_budget)}"
+        else:
```

Comment:
Should we use shared as the fallback, or explicitly check for per-category? There are no formal promises/contracts on audit log data and we could end up with different data overtime.

Reply:
I've swapped the if-else blocks